provider/maas/environ_whitebox_test.go

```diff
@@ -1759,19 +1762,9 @@ func (s *environSuite) TestStartInstanceDistribution(c *gc.C) {
 	s.testMAASObject.TestServer.AddZone("test-available", "description")
 	s.newNode(c, "node1", "host1", map[string]interface{}{"zone": "test-available"})
 	inst, _ := testing.AssertStartInstance(c, env, "1")
-	c.Assert(inst.(*maasInstance).zone(), gc.Equals, "test-available")
-}
-
-func (s *environSuite) TestStartInstanceDistributionAZNotImplemented(c *gc.C) {
```

Comment:
Why drop this? Because we no longer support any version of MAAS that doesn't support AZs?